src/modules/initScoreRefresh.js

```diff
@@ -0,0 +1,9 @@
+const refreshBtn = document.querySelector('#refresh-btn');
+
+const initRefreshBtn = (game) => {
+  refreshBtn.addEventListener('click', () => {
```

Comment:
- [ ] Kindly consider making sure that when the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change. because currently, it's not causing any change fix that to meet the below requirement
> Implement the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).

Reply:
sorry about that @divinecharlotte, I have implemented updation of score list on submitting of new score, that make list up-to-date even before hitting refresh button 😆